DEVELOPING_A_RUNTIME.md

```diff
@@ -16,146 +36,161 @@ A **required** exported constant that decides which version of the Runtime API t
 
 The latest and suggested version is `3`.
 
-### `analyze`
+**Example:**
 
-An **optional** exported function that returns a unique fingerprint used for the purpose of [build de-duplication](https://vercel.com/docs/v2/platform/deployments#deduplication). If the `analyze` function is not supplied, a random fingerprint is assigned to each build.
-
-```js
-export analyze({
-  files: Files,
-  entrypoint: String,
-  workPath: String,
-  config: Object
-}) : String fingerprint
+```typescript
+export const version = 3;
 ```
 
-If you are using TypeScript, you should use the following types:
+### `build()`
 
-```ts
-import { AnalyzeOptions } from '@vercel/build-utils'
+A **required** exported function that returns a [Serverless Function](#serverless-function).
```

Comment:
This anchor doesn't go anywhere? Maybe remove the link?

Reply:
Good catch.